tensorflow/core/kernels/mkl/mkl_dequantize_op.cc

```diff
@@ -53,15 +55,16 @@ class MklDequantizeOp : public OpKernel {
       auto cpu_engine = engine(engine::kind::cpu, 0);
 
       // Get the inputs
-      const Tensor& src_tensor = MklGetInput(ctx, kSrcIndex);
-      const float min_range =
-          MklGetInput(ctx, kMinIndex).template flat<float>()(0);
-      const float max_range =
-          MklGetInput(ctx, kMaxIndex).template flat<float>()(0);
+      const Tensor& src_tensor = ctx->input(kSrcIndex);
+      const float min_range = ctx->input(1).flat<float>()(0);
+      const float max_range = ctx->input(2).flat<float>()(0);
```

Comment:
Can you still use kMinIndex/kMaxIndex?

Reply:
Thanks for the comment. The code has been updated now to use `kMinIndex` and `kMaxIndex`